atlascope/core/rest/endpoints/tile_endpoints.py

```diff
@@ -26,14 +27,11 @@ class TileMetadataView(GenericAPIView, mixins.RetrieveModelMixin):
 
     def get(self, *args, **kwargs):
         dataset = self.get_object()
-        cached = fsspec.open_local(
-            f'simplecache::{dataset.content.url}',
-            filecache={'cache_storage': '/tmp/files'},
-        )
+        content_location = Path(settings.BASE_DIR, dataset.content.name)
```

Comment:
```suggestion
        content_location = dataset.content.path
```

[I think this is given to us by Django under this property.](https://docs.djangoproject.com/en/4.0/ref/models/fields/#django.db.models.fields.files.FieldFile.path)